lib/originals-helper.js

```diff
@@ -56,6 +56,75 @@ function getModuleNameForType(type) {
   }
 }
 
+/**
+ * @param {Node} node - a ts-morph AST node
+ * @param {string} property - the property to find on some module
+ *
+ * @return {string} the get-originals module name corresponding to the node's
+ * type. (This is sometimes different from the TypeScript type name.)
+ */
+function getModuleNameForNodeFromPrototypeChain(node, property) {
```

Comment:
The two functions `getModuleNameForNode` and `getModuleNameForNodeFromPrototypeChain` behave very differently and have rather different usese, despite their similar names. I think we should differentiate them better in name, and in the jsdoc comments.

One idea is `getModuleNameForPropertyOfNode(node, property)` (or maybe even `(property, node)`?).

The jsdoc should explain that it's not the module name corresponding to the Node's type that you're looking for. It's the module name where that property of the node is found.